### PR TITLE
Add RADAI to Data Tools section of README.md

### PR DESCRIPTION
**Pull request by thjoshi**
An entry for RADAI has been added to the README.md. 

This includes a link to the repository and a description of of the library.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -185,3 +185,11 @@ classification of multimodal data to inform nuclear security and proliferation d
 scenarios." It has a modular framework with the ability to pre-process data as well as train
 and test models. MIMOSAS is compatible with [MINOS](MINOS.md) data and can be installed from
 source. Additional information can be found [here](https://complexity.berkeley.edu/mimosas/).
+
+3/ "[RADAI](https://gitlab.com/lbl-anp/radai/radai) (Radiological Anomaly Detection And
```
Comment (jccurtis) [inline README.md]:
```suggestion
3. "[RADAI](https://gitlab.com/lbl-anp/radai/radai) (Radiological Anomaly Detection And
```

Comment (stompsjo) [inline README.md]:
```suggestion
1. "[RADAI](https://gitlab.com/lbl-anp/radai/radai) (Radiological Anomaly Detection And
```

We can leave it at 1. and let Markdown order it for us.

**Review by stompsjo — CHANGES_REQUESTED** [review]
Barring a rebase I think this looks great, thanks for the addition!

**Review by stompsjo — APPROVED** [review]
I took the liberty of editing and rebasing. LGTM! Thanks for the addition!

**Pull request merged**